config/nrfconnect/chip-module/Kconfig.defaults

```diff
@@ -272,6 +276,17 @@ if NET_L2_OPENTHREAD
 config IEEE802154_NRF5_RX_STACK_SIZE
     default 1024
 
+endif # NET_L2_OPENTHREAD
```

Comment:
Should we change line 271:

`default y if !WIFI_NRF70`
to
`default y if OPENTHREAD`